chia/data_layer/data_layer.py

```diff
@@ -568,6 +568,7 @@ async def fetch_and_validate(self, store_id: bytes32) -> None:
         servers_info = await self.data_store.get_available_servers_for_store(store_id, timestamp)
         # TODO: maybe append a random object to the whole DataLayer class?
         random.shuffle(servers_info)
+        success = False
```

Comment:
I think if we change the `break` on success to a `return` then we don't need this to be a 'bigger scoped' variable.